community/kernel/src/main/java/org/neo4j/storageengine/api/StorageEngine.java

```diff
@@ -36,14 +36,18 @@
 public interface StorageEngine
 {
     /**
+     * Creates a new {@link StorageReader} for reading committed data from the underlying storage.
+     * The returned instance is intended to be used by one transaction at a time, although can and should be reused
```

Comment:
This interface should not describe behavior of other components that have nothing to do with it